x/msgfees/keeper/proposal_handler.go

```diff
@@ -38,6 +43,24 @@ func HandleAddMsgFeeProposal(ctx sdk.Context, k Keeper, proposal *types.AddMsgFe
 	return nil
 }
 
+// DetermineBips converts basis point string to uint32
+func DetermineBips(recipient string, recipientBasisPoints string) (uint32, error) {
+	var bips uint32
+	if len(recipientBasisPoints) > 0 && len(recipient) > 0 {
+		bips64, err := strconv.ParseUint(recipientBasisPoints, 10, 32)
+		if err != nil {
+			return bips, err
```

Comment:
Should probably wrap this error with some context indicating that it's the recipient basis points field that didn't parse correctly.
```suggestion
			return bips, types. ErrInvalidBipsValue.Wrap(err.Error())
```

Reply:
Good call.  I also did the same to the error below it when the bips value is greater than 10,000.  Agree I should have?

Reply:
Yup! Looks good!